### ISSUE TITLE
swirl.py not working on some ubuntu configs

### ISSUE THREAD

**Issue by sidprobstein** [sidprobstein/swirl-search#47]
If rabbit becomes 'lost', e.g. 

```
sid@swirl1:/usr/local/swirl$ python swirl.py status
##S#W#I#R#L##1#.#5##############################################################

Service: rabbitmq...UNKNOWN, pid:108968 not found
Service: django...RUNNING, pid:109050
Service: celery-worker...RUNNING, pid:109051
Service: celery-beats...RUNNING, pid:109056

    PID TTY          TIME CMD
 109050 pts/4    00:00:00 daphne
 109051 pts/4    00:00:02 celery
 109056 pts/4    00:00:02 celery

```

Shutdown swirl:

```
python swirl.py stop
```

An error will occur since rabbit can't be found at the original pid. Remove the .pid file. Find rabbit and terminate it, e.g.

```
ps -ef | grep rabbit
...
kill rabbits-new-pid
```

As a workaround: [ start and stop processes manually](https://github.com/sidprobstein/swirl-search/wiki/5.-Admin-Guide#manually-starting-services)

**Comment by sidprobstein** [comment]
Rabbitmq is running on many ubuntus; this is the culprit

**Comment by sidprobstein** [comment]
Improvements TBD in 1.6

**Comment by sidprobstein** [comment]
Fixed in 1.6. swirl.py now checks to see if rabbitmq is running, and skips it if so

**Comment by sidprobstein** [comment]
sid@agentcooper swirl-search-master % python swirl.py start
##S#W#I#R#L##1#.#6##############################################################

Warning: rabbitmq appears to be running, skipping it:
  501 95899 95503   0  1:54PM ttys000    0:00.01 /bin/sh /usr/local/sbin/rabbitmq-server
Start: django -> daphne swirl_server.asgi:application ... Ok, pid: 95948
Start: celery-worker -> celery -A swirl_server worker --loglevel=info ... Ok, pid: 95963
Start: celery-beats -> celery -A swirl_server beat -l INFO --scheduler django_celery_beat.schedulers:DatabaseScheduler ... Ok, pid: 95994
Updating .swirl... Ok

  PID TTY           TIME CMD
95948 ttys002    0:00.68 /Library/Frameworks/Python.framework/Versions/3.10/Resources/Python.app/Contents/MacOS/Python /Library/Frameworks/Python.framework/Versions/3.10/bin/daphne swirl_server.asgi:application
95963 ttys002    0:05.21 /Library/Frameworks/Python.framework/Versions/3.10/Resources/Python.app/Contents/MacOS/Python /Library/Frameworks/Python.framework/Versions/3.10/bin/celery -A swirl_server worker --loglevel=info
95994 ttys002    0:03.92 /Library/Frameworks/Python.framework/Versions/3.10/Resources/Python.app/Contents/MacOS/Python /Library/Frameworks/Python.framework/Versions/3.10/bin/celery -A swirl_server beat -l INFO --scheduler django_celery_beat.schedulers:DatabaseScheduler

Command successful!
